src/libbson/src/bson/bson-utf8.c

```diff
@@ -245,7 +245,7 @@ bson_utf8_validate (const char *utf8, /* IN */
  *       two byte UTF-8 sequence.
  *
  * Parameters:
- *       @utf8: A UTF-8 encoded string.
+ *       @utf8: A validated UTF-8 encoded string.
```

Comment:
```suggestion
 *       @utf8: A UTF-8 encoded string.
```
Adding a validity requirement is a breaking change given the function is written to handle [invalid inputs](https://github.com/mongodb/mongo-c-driver/blob/8a85033b212dabf140442dd094f636ef2d44b8e5/src/libbson/src/bson/bson-utf8.c#L320). See other comment regarding "could not be escaped".